### ISSUE TITLE
lets move discussions to nostr

### ISSUE THREAD

**Issue by PeerRich** [NostrGit/NostrGit#169]
why dont we open a new page:

nostrgit.com/discussions

that hosts a single thread (written by the nostrgit account)

and people can reply to it

that way we have a nostr-first place for "general" chat

@cypherhoodlum can you post a generic "discuss nostr" post and send the event here?

**Comment by PeerRich** [comment]
ive seen this on https://nostrocket.org and love the idea

**Comment by cypherhoodlum** [comment]
Good idea :+1: 
However, I think I'm just hours away from mining the vanity address npub1n0strg7t and I'd like to migrate the account to that npub before establishing the nostr profile more publicly. We just have to wait a bit more! :smile:

**Comment by PeerRich** [comment]
oh perfect! can we try one of the multi-access tools so I can post on that new nostr account too? maybe we also find a tool that syncs with twitter so we can post on both platforms: twitter.com/nostrgit at the same time

**Comment by PeerRich** [comment]
btw curious how do you migrate an account? or does that just mean deleting/telling everyone to follow the new account?

**Comment by cypherhoodlum** [comment]
> btw curious how do you migrate an account? or does that just mean deleting/telling everyone to follow the new account?

Yeah just migrating conceptually, not functionally. Edit the name to something like "deleted account" and just make a post "here's the new npub of the NostrGit profile" or something similar. And edit the shorthand links on our GitHub.

> can we try one of the multi-access tools so I can post on that new nostr account too?

Did we find a tool we can use for that? I'm all for it. I can also just give you the private key if we don't have a better solution right now. Here's the key: nsec1m2yul9kmsvs9vqglj9zn2wcmecyafvdtu6uc9ptr4egxgr7l3feqam2n7v
Just kidding. Not the real key :grin:

**Comment by PeerRich** [comment]
opened a PR #173 importing the comments of a thread, pretty nice

**Comment by PeerRich** [comment]
> edit: but in all seriousness, I can give you the key as soon as I mine it.

sure! i think it would be awesome to have the key in some sort of multi-access tool so we can eventually give more people access to post without giving up the nsec

**Comment by e-roy** [comment]
> I think PRs may have to be a bit different from the rest, and the focus for now should be on Issues & Discussions. PRs are so intimately git related (backend stuff). The `kind 30023` notes seem a bit overkill to me. I would stick with `kind 1` notes. Issues, and especially Discussion notes should be much more concise than a blog post. One Discussion thread may get convoluted very fast, but I like the idea of having one general Discussion thread from which more formalised Issues emerge. In summary, I think we could try to have one general Discussion thread. Issues would be more specifically related to a bug, feature, etc.

I was thinking of all 3 being similar in page navigation, ref #150 

`kind 30023` does seems to allow markdown for content, which is what we are using on this page.  

I've been exploring both of these  and created a little app to understand more about it.  [https://nostr-app.vercel.app/](https://nostr-app.vercel.app/).  Here is the repo [https://github.com/e-roy/nostr-app](https://github.com/e-roy/nostr-app)

here is the page for that I've been starting to test  `kind 1` notes on [https://nostr-app.vercel.app/notes/note10488mc62vzcf7yw8x0pavewdh5vmm5j4r279l9x43rss8tjzdc0s2lsrfp](https://nostr-app.vercel.app/notes/note10488mc62vzcf7yw8x0pavewdh5vmm5j4r279l9x43rss8tjzdc0s2lsrfp)

As far as someone raising an issue or initiating a discussion, I think we could make use of the tags and tags can be used during querying for results.  It's still a theory, but the idea would be create a tag with the repo `id` and any time some creates a issue or initiates a discussion, the tag is filled with that `id` and then could be used in the filter to get the proper results for that particular repo.

I'm still testing out ideas so I can get a better understanding of the nostr protocol.  so much to learn!  😄

**Comment by cypherhoodlum** [comment]
> kind 30023 does seems to allow markdown for content

Okay that's a huge plus. Honestly I have nothing against any working solution. These are just nuances.
Looks promising! :rocket:

**Comment by casperfriendlyghost6** [comment]
I would like to create temporary discussions, after a while they are deleted.

![image](https://user-images.githubusercontent.com/131043818/232613210-70adaffa-a141-4fcc-990b-8ba26ac5d0eb.png)

**Comment by cypherhoodlum** [comment]
> I would like to create temporary discussions, after a while they are deleted.

There are not many relays that enable deleting notes. I think all Discussions should happen on the nostr network for easier aggregation on the NostrGit clients, and not be stored/hosted elsewhere. That being said, I don't see how Discussion expiration could be a realistic functionality. Or do you mean older events would be not fetched anymore at some point, making them "hidden"?

**Comment by cypherhoodlum** [comment]
I think [NIP-28](https://github.com/nostr-protocol/nips/blob/master/28.md) is worth exploring when figuring out how Discussions should work on NostrGit.
